build_tools/travis/test_script.sh

```diff
@@ -38,6 +38,12 @@ run_tests() {
     # disk caching does not work.
     export SKLEARN_SKIP_NETWORK_TESTS=1
 
+    if [[ "$DISTRIB" == "scipy-dev-wheels" ]]; then
+        # Do not run doctests in scipy-dev-wheels build for now
+        # (broken by numpy 1.14.dev array repr/str formatting change)
+        export PYTEST_ADDOPTS="-p no:doctest"
```

Comment:
I'd rather modify `TEST_CMD` than mix local and global env changes.

Reply:
I ended up using `np.set_printoptions(sign='legacy')` in conftest.py. This way, the Travis test_script.sh code is simpler and it works locally for people using numpy > 1.13.